plugins/oauth/server/providers/globus.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+import struct
+import jwt
+from jwt.utils import base64url_decode
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.backends import default_backend
+from six.moves import urllib
+from six import string_types
+
+from girder.api.rest import getApiUrl, RestException
+from .base import ProviderBase
+from .. import constants
+
+
+def decode_value(val):
```

Comment:
Since this is a public symbol, let's make it camelCase and also add a docstring. (Or just make it private if you prefer.)